crates/core/tedge_api/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "tedge_api"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.53"
+async-trait = "0.1.52"
+thiserror = "1.0.30"
+tokio = { version = "1.16.1", features = ["sync"] }
+toml = "0.5.8"
+uuid = { version = "0.8", features = ["serde", "v4"] }
```

Comment:
Just a small technical detail, but I believe there is no need to use `uuid`, if you really need a crypto safe id rather go for `nanoid`, should be a bit faster and fills in the same role.